app/views/users/passwords/edit.html.slim

```diff
@@ -17,7 +17,7 @@
         = f.input :password_confirmation, required: true
 
       .form-actions
-        = f.button :submit
+        = f.button :submit, 'Update password'
```

Comment:
@timurvafin because of this http://take.ms/yL3Qy

"Sign up" button on change password page